config.ts

```diff
@@ -36,8 +37,11 @@ const paths = {
   docsDist: base.bind(null, envConfig.dir_docs_dist),
   docsSrc: base.bind(null, envConfig.dir_docs_src),
   umdDist: base.bind(null, envConfig.dir_umd_dist),
+  posix: undefined, // all the sibling values, but with forward slashes regardless the OS
 }
 
+paths.posix = _.mapValues(paths, func => (...args) => func(...args).replace(/\\/g, '/'))
```

Comment:
seems that we should consider it to be a mandatory preprocessing method for provided values, so that for `paths` we would have POSIX ones by default:

```
paths = posix({
   src: ...,
   dist: ...,
   ...
})

// and then
paths.src()    // returns POSIX variant by default

// and, if it will be necessary, we can do the following (this will be about extending functionality of 'posix()')
paths.original.src()
```

The reason I see it being a better approach is that we will be provided with a safe defaults as a result - in contrast to the approach that is currently suggested. What do you think?